Source/Scene/ClassificationModel.js

```diff
@@ -0,0 +1,1807 @@
+            get : function() {
+                if (!defined(this._extensionsRequired)) {
+                    this._extensionsRequired = getRequiredExtensions(this);
+                }
+                return this._extensionsRequired;
+            }
+        },
+
+        /**
+         * Gets the model's up-axis.
+         * By default models are y-up according to the glTF spec, however geo-referenced models will typically be z-up.
+         *
+         * @memberof ClassificationModel.prototype
+         *
+         * @type {Number}
+         * @default Axis.Y
+         * @readonly
+         *
+         * @private
+         */
+        upAxis : {
+            get : function() {
+                return this._upAxis;
+            }
+        },
+
+        /**
+         * Gets the model's triangle count.
+         *
+         * @private
+         */
+        trianglesLength : {
+            get : function() {
+                return this._trianglesLength;
+            }
+        },
+
+        /**
+         * Gets the model's geometry memory in bytes. This includes all vertex and index buffers.
+         *
+         * @private
+         */
+        geometryByteLength : {
+            get : function() {
+                return this._geometryByteLength;
+            }
+        },
+
+        /**
+         * Gets the model's texture memory in bytes.
+         *
+         * @private
+         */
+        texturesByteLength : {
+            get : function() {
+                return 0;
+            }
+        },
+
+        /**
+         * Gets the model's classification type.
+         * @memberof ClassificationModel.prototype
+         * @type {ClassificationType}
+         */
+        classificationType : {
+            get : function() {
+                return this._classificationType;
+            }
+        }
+    });
+
+    /**
+     * This function differs from the normal subarray function
+     * because it takes offset and length, rather than begin and end.
+     */
+    function getSubarray(array, offset, length) {
+        return array.subarray(offset, offset + length);
+    }
+
+    var aMinScratch = new Cartesian3();
+    var aMaxScratch = new Cartesian3();
+
+    function getAccessorMinMax(gltf, accessorId) {
+        var accessor = gltf.accessors[accessorId];
+        var extensions = accessor.extensions;
+        var accessorMin = accessor.min;
+        var accessorMax = accessor.max;
+        // If this accessor is quantized, we should use the decoded min and max
+        if (defined(extensions)) {
+            var quantizedAttributes = extensions.WEB3D_quantized_attributes;
+            if (defined(quantizedAttributes)) {
+                accessorMin = quantizedAttributes.decodedMin;
+                accessorMax = quantizedAttributes.decodedMax;
+            }
+        }
+        return {
+            min : accessorMin,
+            max : accessorMax
+        };
+    }
+
+    function computeBoundingSphere(model) {
```

Comment:
Are you going to move some of these functions to a common file?  Or do you want to do that in a separate PR?